build/getDependencies-windows.sh

```diff
@@ -109,7 +109,7 @@ rm -rf ../src/BloomBrowserUI/bookEdit/test/libsynphony
 #     clean: false
 #     revision: latest.lastSuccessful
 #     paths: {"Palaso.BuildTasks.dll"=>"build/", "*.dll"=>"lib/dotnet"}
-#     VCS: https://github.com/sillsdev/libpalaso.git [NoStrongName]
```

Comment:
Why is this being changed? It caused all kinds of problems when we used the strong-name version of libpalaso.